dspace/config/modules/stats.cfg

```diff
@@ -0,0 +1,35 @@
+# Enable the IRUS tracker. By default or when omitted, the tracker will be disabled
+stats.tracker.enabled = false
```

Comment:
Since this entire file is specific to IRUS, we should rename it to be `irus-statistics.cfg` and prefix all settings with `irus.statistics.tracker` (instead of `stats.tracker`).

This would bring it more in line with the other stats config files: `usage-statistics.cfg` and `solr-statistics.cfg`.  Naming this just `stats.cfg` would likely be confusing to users.

Reply:
I updated the variables and changed the IRUS property names to contain IRUS and be less generic